### ISSUE TITLE
Rest platform headers not working

### ISSUE THREAD

**Issue by Aerion** [rmountjoy92/DashMachine#90]
Rest platform headers are interpreted as string instead of being interpreted as dictionary.
I didn't manage to get the example from the documentation to work.

To reproduce, use the following config (from example, just added the `headers` line from the rest doc).

```ini
[test]
platform = rest
resource = https://pokeapi.co/api/v2/pokemon
value_template = Pokemon: {{value}}
headers = {"Content-Type": "application/json"}

[Pokemon]
prefix = https://
url = pokemon.com
icon = static/images/apps/default.png
description = Data sources example
open_in = this_tab
data_sources = test
```

You'd expect a json response, but here is the output of `load_data_source?id=<this_id>`:
```
Pokemon: 'str' object has no attribute 'items'
```

That is because of the `self.headers` value passed here is of type `str` while requests.get expects a `dict`. https://github.com/rmountjoy92/DashMachine/blob/ec082dae46ad24fbe821c98b2cfb3e30ee12fbfe/dashmachine/platform/rest.py#L87-L89

**Comment by Aerion** [comment]
After the fix in #91, you have the following expected result:

```
Pokemon: {'count': 964, 'next': 'https://pokeapi.co/api/v2/pokemon?offset=20&limit=20', 'previous': None, 'results': [{'name': 'bulbasaur', 'url': 'https://pokeapi.co/api/v2/pokemon/1/'}, {'name': 'ivysaur', 'url': 'https://pokeapi.co/api/v2/pokemon/2/'}, {'name': 'venusaur', 'url': 'https://pokeapi.co/api/v2/pokemon/3/'}, {'name': 'charmander', 'url': 'https://pokeapi.co/api/v2/pokemon/4/'}, {'name': 'charmeleon', 'url': 'https://pokeapi.co/api/v2/pokemon/5/'}, {'name': 'charizard', 'url': 'https://pokeapi.co/api/v2/pokemon/6/'}, {'name': 'squirtle', 'url': 'https://pokeapi.co/api/v2/pokemon/7/'}, {'name': 'wartortle', 'url': 'https://pokeapi.co/api/v2/pokemon/8/'}, {'name': 'blastoise', 'url': 'https://pokeapi.co/api/v2/pokemon/9/'}, {'name': 'caterpie', 'url': 'https://pokeapi.co/api/v2/pokemon/10/'}, {'name': 'metapod', 'url': 'https://pokeapi.co/api/v2/pokemon/11/'}, {'name': 'butterfree', 'url': 'https://pokeapi.co/api/v2/pokemon/12/'}, {'name': 'weedle', 'url': 'https://pokeapi.co/api/v2/pokemon/13/'}, {'name': 'kakuna', 'url': 'https://pokeapi.co/api/v2/pokemon/14/'}, {'name': 'beedrill', 'url': 'https://pokeapi.co/api/v2/pokemon/15/'}, {'name': 'pidgey', 'url': 'https://pokeapi.co/api/v2/pokemon/16/'}, {'name': 'pidgeotto', 'url': 'https://pokeapi.co/api/v2/pokemon/17/'}, {'name': 'pidgeot', 'url': 'https://pokeapi.co/api/v2/pokemon/18/'}, {'name': 'rattata', 'url': 'https://pokeapi.co/api/v2/pokemon/19/'}, {'name': 'raticate', 'url': 'https://pokeapi.co/api/v2/pokemon/20/'}]}
```

**Comment by rmountjoy92** [comment]
fixed in https://github.com/rmountjoy92/DashMachine/pull/91
